Sources/Internal/OperationsManager.swift

```diff
@@ -18,6 +18,18 @@ import Foundation
 class OperationsManager {
   private var grpcClient: GrpcClient
 
+  private let queryRefAccessQ = DispatchQueue(
+    label: "firebase.dataconnect.queryRef.AccessQ",
+    autoreleaseFrequency: .workItem
+  )
+  private var queryRefs = [AnyHashable: any ObservableQueryRef]()
```

Comment:
Can `queryRefs` and `mutationRefs` be declared with `let` instead of `var`?

Reply:
No since let won't allow modification of the dictionary.